client/packages/common/src/plugins/types.ts

```diff
@@ -28,17 +29,24 @@ export type Plugins = {
   requestRequisitionLine?: {
     tableStateLoader: React.ComponentType<{
       requestLines: RequestLineFragment[];
+      requisition: RequestFragment;
     }>[];
     tableColumn: ColumnDef<RequestLineFragment>[];
     editViewField: React.ComponentType<{
       line: RequestLineFragment;
+      draft?: DraftRequestLine;
       unitName?: string;
     }>[];
     editViewInfo: React.ComponentType<{
       line: RequestLineFragment;
       requisition: RequestFragment;
     }>[];
   };
```

Comment:
Can you change this to be like the other types please.

requestRequisition {
    sidePanelSection
}